testsuite/testsuite-opentracing/src/test/java/org/wildfly/swarm/opentracing/SkipPatternTest.java

```diff
@@ -42,7 +40,7 @@
  * @author Juraci Paixão Kröhling
  */
 @RunWith(Arquillian.class)
-public class OpenTracingWithSkipPatternTest {
+public class SkipPatternTest {
```

Comment:
I changed the name of the tests to be more concise. If it's not appropriate for this PR, I can revert it.